documentation/src/main/asciidoc/topics/proc_enabling_jmx.adoc

```diff
@@ -34,6 +36,17 @@ GlobalConfiguration globalConfig = new GlobalConfigurationBuilder()
 
 <1> registers {brandname} JMX MBeans.
 
+.JMX Java Client
+In order to retrieve or do operations in the server you can create your own Java Client:
```

Comment:
Do we really want to document how to create a JMX client in Java? IMO this is beyond the scope of the Infinispan documentation.

Reply:
Well, I think that it is easy for developers to have the example ready instead of searching in the internet how to do that. The dev can read the doc, copy and paste and it is ready.

Reply:
There has to be a limit on what we do/don't include. I also suspect that a lot of users would be using some form of existing tooling to interact with MBeans. @oraNod any thoughts?

Reply:
I see a couple things here. One is that we're mixing things up a little here. The topic for enabling JMX is specifically for configuring Infinispan, which can be either embedded or server. So I think that if we do add this detail about creating a JMX client for server, it should be in its own procedural topic. That means we can put it directly in the Server Guide.

I also think this needs a bit more context and clearer set of steps, so a separate procedure is necessary if we do go ahead and add this content.

So far as the limit on what we do/don't include in the docs. I agree that this falls within the realm of 3rd party which we should generally avoid because it does add maintenance. On the community side this could be a helpful tip or bit of info that could fall into monitoring and troubleshooting. On the enterprise side we should not include 3rd party stuff in the docs to avoid support or security concerns.

That said, I don't want to shoot anyone down. @diegolovison if you really feel like this is valuable for the docs then I'd suggest creating a new procedure and filling it out a little more. We can then take a look and better assess where it should fit in the doc set.

Reply:
I think we can remove the code: `avoid support or security concerns` are good points.